en/docs/tutorials/create-your-first-rest-api.md

```diff
@@ -52,78 +56,81 @@ The `Statistics` REST API opens on a separate page.
+1. To open the REST API component in the Web Editor, click **Edit Code**.
     
     !!! info
-        Opening the VS Code Editor may take a little while if you are a first-time user.
+        Opening the Web Editor may take a little while if you are a first-time user.
+
+2. Remove the resource that is configured by default for the REST API by by clicking its **Delete** icon.
 
-2. Add a new construct and an API resource as follows:
+   ![Delete resource](../assets/img/tutorials/rest-api/delete-resource.png){.cInlineImage-half}
 
-    1. Click on the first **+** icon to add an HTTP construct. Select the **Service** construct type,
+3. Add a new resource by clicking the **+** icon under the existing construct.
 
-        ![Add service construct](../assets/img/tutorials/rest-api/add-service-construct.png){.cInlineImage-half}
+   ![Add resource](../assets/img/tutorials/rest-api/add-resource.png){.cInlineImage-half}
 
-        Select HTTP as the service type, and enter the following information.
+    Then click **Resource**. 
 
-        | **Field**             | **Value**             |
-        |-----------------------|-----------------------|
-        | **Service Base Path** | `COVID19-Statistics`  |
-        | **Listener Port**     | `9090`                |
+    As a result, a panel named **Configure Resource** appears on the right of the page. In this panel, enter information as follows:
 
-        Select the **Define Inline** check box, and then click **Save**.
+     | **Field**       | **Value**                |
+     |-----------------|--------------------------|
+     | **HTTP Method** | `GET`                    |
+     | **Path**        | `stats/[string country]` |
+     | **Return Type** | `json|error?`            |
 
-    2. Expand the construct you created by clicking on it. Then click the **EDIT** icon of the default `GET` resource within the construct.
+    Here, you are configuring a GET resource to fetch statistics related to COVID-19.
 
-    3. Enter `stats/[string country]` in the **Path** field.
+    By adding `string country` within square brackets in the path, you are introducing `country` as a path parameter for which the value should be in the string format.
 
-        By adding `string country` within square brackets in the path, you are introducing `country` as a path parameter for which the value should be in the string format.
+    The specified return type allows the REST API to return the output in either JSON or error? format.
 
-    4. Click **Advanced** and in the **Return Type** field, enter `json|error` to get the return in `json` or `error1` format.
+    Click **Save**.
 
-    5. Click **Save API**.
+    The low-code diagram of this resource opens .
 
-3. To connect to the COVID-19 API and retrieve data, follow the sub-steps below.
+4. To connect to the COVID-19 API and retrieve data, follow the sub-steps below.
```

Comment:
We need a colon after below right? Doesn't seem like the suggestion has been applied.

Reply:
Done.